libs/video-view/src/index.tsx

```diff
@@ -1,40 +1,36 @@
-import { Box, Center, Flex, Spinner, Stack } from '@chakra-ui/react';
+import { Box, Center, Flex, HStack, Spinner } from '@chakra-ui/react';
 import React, { memo, useEffect, useRef, useState } from 'react';
 
-import ControlBar from '@millicast-react/control-bar';
-import { IconCameraOff, IconCameraOn, IconMicrophoneOff, IconMicrophoneOn } from '@millicast-react/dolbyio-icons';
 import InfoLabel from '@millicast-react/info-label';
 
-import { VideoViewProps } from './types';
+import { HTMLVideoElementWithCaptureStream, VideoViewProps } from './types';
 
 const VideoView = ({
-  // displayFullscreenButton = true,
   displayVideo = true,
   height,
   label,
   maxHeight,
   maxWidth,
   mediaStream,
-  mirrored = false,
+  mirrored,
   muted = true, // Has to be true for AutoPlay in chromium
   onClick,
   onError,
   onSrcMediaStreamClose,
   onSrcMediaStreamReady,
   placeholderNode,
+  playing = true,
```

Comment:
What's the intention of this property?  It sounds like a state instead of the initial property. Do you mean `autoPlay` here?

Reply:
`playing` is externally controlled video playback state. Its corresponding `useEffect` updates the video's internal state.

Reply:
OK, if it's a property that is set by the caller, we'd better name it `shouldPlay`, it's a kind of request, am I correct?